acra-censor/acra-censor_test.go

```diff
@@ -855,3 +853,36 @@ func testSyntax(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestDifferentTablesParsing(t *testing.T) {
+	testQuery := "SELECT Orders.OrderID, Customers.CustomerName, Shippers.ShipperName FROM ((Orders INNER JOIN Customers ON Orders.CustomerID = Customers.CustomerID) INNER JOIN Shippers ON Orders.ShipperID = Shippers.ShipperID);"
```

Comment:
hard to read such query, first time see parenthesized joins in such way ;). can you reformat to smth like:
```
testQuery := `
    SELECT Orders.OrderID, Customers.CustomerName, Shippers.ShipperName 
    FROM Orders 
    INNER JOIN Customers ON Orders.CustomerID = Customers.CustomerID
    INNER JOIN Shippers ON Orders.ShipperID = Shippers.ShipperID;
`
```
some sql style guides:
* https://gist.github.com/fredbenenson/7bb92718e19138c20591
* https://www.sqlstyle.guide/
* https://medium.com/@lenguyenthedat/sql-style-guide-9d2d1ae805d2

Reply:
Done